app/javascript/components/App.js

```diff
@@ -11,6 +11,7 @@ import PatientInfo from "./pages/PatientInfo";
 import PatientNew from "./pages/PatientNew";
 import PatientEdit from "./pages/PatientEdit";
 import "./App.css";
+import { resolveConfig } from "prettier";
```

Comment:
I've never seen this before. what do?

Reply:
oh hmm, i didn't catch that, i don't know if it actually does anything, i'll investigate